transmission_rpc/client.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         username: str | None = None,
         password: str | None = None,
         host: str = "127.0.0.1",
-        port: int = 9091,
+        port: int | None = 9091,
```

Comment:
None is not handled in `__init__`

Reply:
Done for the URL string generation. `HTTPConnectionPool` and `HTTPSConnectionPool` do accept `None` and will use 80 and 443 in that case so no need to address those.